ui/tests/integration/components/search-select-with-modal-test.js

```diff
@@ -0,0 +1,132 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { create } from 'ember-cli-page-object';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import { clickTrigger, typeInSearch } from 'ember-power-select/test-support/helpers';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import ss from 'vault/tests/pages/components/search-select';
+
+const component = create(ss);
+
+module('Integration | Component | search select with modal', function (hooks) {
```

Comment:
Might be nice to make sure the fallback functionality works for future-proofing

Reply:
done! thank you!